src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -634,6 +634,9 @@ void setRenamePrefixNamespaceAssumeCrossModuleNames(boolean assume) {
   /** Flattens multi-level property names (e.g. a$b = x) */
   public boolean collapseProperties;
 
+  /** Flattens property names created by module rewriting (e.g. a$b = x) */
+  public boolean collapseModuleExportProperties;
```

Comment:
I'd prefer to make collapseProperties an enum over adding a new boolean for this.  I just sent out a refactoring to make the regular collapseProperties private to enable you to change it here.

Reply:
I actually waffled back and forth on this. It was just easier to do it this way. That refactor will help though.

Reply:
Ok, pushed my change out in https://github.com/google/closure-compiler/commit/36544c6f941fdb860eadfeedc6a932dbc28b8ea2